### PR TITLE
Migrate to NUnit 4.1.0

### PR DESCRIPTION
**Pull request by akuegel**
To migrate to NUnit 4.1.0 and NUnit3TestAdapter 4.5.0, the test expectations need to be rewritten. Instead of `Assert.AreEqual(expected, actual)` we will have `Assert.That(actual, Is.EqualTo(expected)`.

Also target .net framework 4.8

### CODE REVIEW

**Comment by akuegel** [comment]
@trullock Not sure if you would be interested in this migration? I am a hobby developer of https://www.yucata.de and we use NUglify to minify our code. We have noticed some small problems with valid css that the parser cannot handle, I would be willing to contribute fixes. I started off with trying to get everything running on my system, and I don't have the old 4.6.1 target framework installed for example, and wanted to use the newer NUnit version for testing.
If you prefer staying on the old version, would you still be open to receive some smaller PRs from me with fixes for the css parser?

File: src/NUglifyApp/App.config
Comment (trullock) [inline src/NUglifyApp/App.config:1]: This looks like a breaking change

**Review by trullock — COMMENTED** [review]
Upgrading the tests is fine, but the modification to NuglifyApp looks breaking. I don't know who cares about this though tbh.

**Comment by trullock** [comment]
> @trullock Not sure if you would be interested in this migration? I am a hobby developer of https://www.yucata.de and we use NUglify to minify our code. We have noticed some small problems with valid css that the parser cannot handle, I would be willing to contribute fixes. I started off with trying to get everything running on my system, and I don't have the old 4.6.1 target framework installed for example, and wanted to use the newer NUnit version for testing. If you prefer staying on the old version, would you still be open to receive some smaller PRs from me with fixes for the css parser?

All PRs welcome, this project needs them.

CSS and JS specs are advancing faster than this project can keep up. I'm the only maintainer who adopted this a few years ago after submitting my own patches. I don't even use this any more so I dont have my own motivation to keep working on it

File: src/NUglifyApp/App.config
Comment (akuegel) [reply src/NUglifyApp/App.config:1]: I have reverted it. I think this got automatically changed when I accepted the suggestion of Visual Studio to target version 4.8.

**Comment by akuegel** [comment]
> > @trullock Not sure if you would be interested in this migration? I am a hobby developer of https://www.yucata.de and we use NUglify to minify our code. We have noticed some small problems with valid css that the parser cannot handle, I would be willing to contribute fixes. I started off with trying to get everything running on my system, and I don't have the old 4.6.1 target framework installed for example, and wanted to use the newer NUnit version for testing. If you prefer staying on the old version, would you still be open to receive some smaller PRs from me with fixes for the css parser?
> 
> All PRs welcome, this project needs them.
> 
> CSS and JS specs are advancing faster than this project can keep up. I'm the only maintainer who adopted this a few years ago after submitting my own patches. I don't even use this any more so I dont have my own motivation to keep working on it

Thanks, good to know, I will work on the two issues we have faced so far. Everything else worked fine, thanks for your work on this :)

**Comment by akuegel** [comment]
Anything else I should change?

**Pull request merged**